### ISSUE TITLE
Tables - changing position of harmonic and spectral noise issue

### ISSUE THREAD

**Issue by tastez** [IggyLabs/IggyLabsModules#39]
trying it out in Rack v2 and the app closes when scanning through harmonic and spectral noise - maybe those  wavetables aren't loaded correctly?

**Comment by isabelgk** [comment]
Yep, I can reproduce. I know something changed with filepath handling with v2 so it may be related...
